docs/UserGuide.adoc

```diff
@@ -217,6 +217,11 @@ Format: `markAssignment INDEX id/ASSIGNMENT_INDEX m/MARK`
 Display the statistics for the specified assignment. +
 Format: `assignmentStats INDEX`
 
+=== Display the statistics for an assignment: `classStats`
+
+Display the statistics for the overall grades of students. +
```

Comment:
Maybe add more details? What will be printed out... Also so that it can be tested